oak/server/rust/oak_abi/src/lib.rs

```diff
@@ -21,6 +21,11 @@ pub use proto::oak_api::OakStatus;
 
 pub mod proto;
 
+/// Handle used to identify read or write channel halves.
+///
+/// These handles are used for all host function calls.
+pub type Handle = u64;
```

Comment:
(cannot comment on line 37 because GitHub): Change type of `INVALID_HANDLE` to `Handle`?

Reply:
Done.